account_banking_pain_base/models/payment_line.py

```diff
@@ -20,33 +20,23 @@
 #
 ##############################################################################
 
-from openerp.osv import orm, fields
+from openerp import models, fields
 
 
-class PaymentLine(orm.Model):
+class PaymentLine(models.Model):
     _inherit = 'payment.line'
 
-    def _get_struct_communication_types(self, cr, uid, context=None):
+    def _get_struct_communication_types(self):
```

Comment:
@pedrobaeza that's true but we need to preserve a stable API for downstream extensions. So these kind a changes should be postponed to a next major upgrade.